### PR TITLE
Add textures, along with temporal accumulation 

### PR DESCRIPTION
**Pull request by LegendWasTaken**
This pull request will add a new texture buffer for temporal accumulation. (And add temporal accumulation)

### CODE REVIEW

File: assets/shaders/compute_pass.comp [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 layout(local_size_x = 16, local_size_y = 16) in;
 layout(binding = 0, rgba8) uniform writeonly image2D result_image;
+layout(binding = 7, rgba8) uniform image2D temporal_image;
```
Comment (cdgiessen) [inline assets/shaders/compute_pass.comp:10]: Why is this not in an order? seems odd to go from 0 to 7 to 1

File: src/rvpt/rvpt.cpp [hunk 2]
```diff
@@ -395,9 +408,11 @@ void RVPT::create_framebuffers()
 RVPT::RenderingResources RVPT::create_rendering_resources()
 {
     std::vector<VkDescriptorSetLayoutBinding> layout_bindings = {
-        {0, VK_DESCRIPTOR_TYPE_COMBINED_IMAGE_SAMPLER, 1, VK_SHADER_STAGE_FRAGMENT_BIT, nullptr}};
+        {0, VK_DESCRIPTOR_TYPE_COMBINED_IMAGE_SAMPLER, 1, VK_SHADER_STAGE_FRAGMENT_BIT, nullptr},
+        {7, VK_DESCRIPTOR_TYPE_STORAGE_IMAGE, 1, VK_SHADER_STAGE_FRAGMENT_BIT, nullptr}
```
Comment (cdgiessen) [inline src/rvpt/rvpt.cpp:412]: Why is this here? this is just for the fullscreen quad, not the intermediate accumulation buffer.

File: assets/shaders/compute_pass.comp (hunk 1 above)
Comment (LegendWasTaken) [reply assets/shaders/compute_pass.comp:10]: I personally believe it makes more sense to have the same types, closer together. (As this also reflects the shader variable order)

Comment (cdgiessen) [reply assets/shaders/compute_pass.comp:10]: Then just reorder the other bindings, thats much clearer.

File: src/rvpt/rvpt.cpp (hunk 2 above)
Comment (LegendWasTaken) [reply src/rvpt/rvpt.cpp:412]: I don't know.

File: assets/shaders/compute_pass.comp (hunk 1 above)
Comment (LegendWasTaken) [reply assets/shaders/compute_pass.comp]: okay did it

File: src/rvpt/rvpt.cpp (hunk 2 above)
Comment (LegendWasTaken) [reply src/rvpt/rvpt.cpp]: Resolved this in the latest commit
